week2/2-website/css/style.css

```diff
@@ -1,19 +1,261 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+*{
 
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+}
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    height: 100vh;
+}
+
+
+.devices{
```

Comment:
Since this is an unordered list. You want to remove the list style which adds the dots. So what you'd want is `list-style-type: none;`

![Screenshot 2020-06-07 at 21 24 16](https://user-images.githubusercontent.com/42983395/83979422-0a8afc00-a906-11ea-9b3b-46d7d0f59da2.png)